### ISSUE TITLE
firewalld dbus issues

### ISSUE THREAD

**Issue by apollo13** [t-woerner/firewalld#135]
Getting:
```
firewall-cmd --state
ERROR:dbus.proxies:Introspect error on :1.277:/org/fedoraproject/FirewallD1: dbus.exceptions.DBusException: org.fedoraproject.FirewallD1.Exception: 'FirewallD' object has no attribute 'path'
Error: 'FirewallD' object has no attribute 'fw'
```
on 0.4.3.1 (same for all other commands). Is there any debug information you need?

**Comment by apollo13** [comment]
Oh, installing ebtables fixed it.

**Comment by t-woerner** [comment]
This has been fixed with pull request #134
